scripts/widget.py

```diff
@@ -36,10 +36,10 @@ def summarize_scenario(
     # Run the simulation with vaccination
     result = simulate_scenario(params, distributions_as_percents=True)
 
-    st.header(f"**{params['scenario_title']}**")
+    st.header(f"*{params['scenario_title']}*")
 
-    prop_vax_help = f"Vaccination of 100% does not guarantee complete immunity if VE is less than 1. VE is {params['ve']}"
-    st.subheader("Percent of each group vaccinated:", help=prop_vax_help)
+    prop_vax_help = f"Based on allocated doses, what percent of each group is successfully immune? In the counter factual scenario, we assume no vaccines are administered. Vaccination of 100% does not guarantee complete immunity if VE is less than 1. VE is {params['ve']}"
```

Comment:
I'm happy for us to report successful vaccinations but that is not what is currently being computed.

E.g. with default of 500k core getting 250k vaccines at VE 0.74, this reports 50% vaccination.